projects/ngx-google-analytics/package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "ngx-google-analytics",
-  "version": "13.0.0",
+  "version": "13.0.1",
   "description": "A simple ng-9 wrapper to load Google Analytics dependency by angular way",
   "keywords": [ "google", "google analytics", "angular", "angular-9+", "typescript" ],
   "homepage": "https://github.com/maxandriani/ngx-google-analytics",
   "license": "MIT",
   "dependencies": {
     "tslib": "^2.0.0"
   },
+  "peerDependencies": {
+    "@angular/common": "^12.0.0",
```

Comment:
This version range doesn't include v13. I would recommend using the following:
```json
  "peerDependencies": {
    "@angular/common": ">=12.0.0",
    "@angular/core": ">=12.0.0"
  }
```
or if you want to be more restrictive/precise, you could go with the following:
```json
  "peerDependencies": {
    "@angular/common": ">=12.0.0 <14.0.0",
    "@angular/core": ">=12.0.0 <14.0.0"
  }
```
See [semver calculator](https://semver.npmjs.com/) for for more info.

Reply:
Fixed it. I decided to be a bit more precise on this, like your suggestion